hathor/sysctl/runner.py

```diff
@@ -76,6 +83,9 @@ def deserialize(self, value_str: str) -> Any:
         if len(value_str) == 0:
             return ()
 
+        if re.match(array_pattern, value_str):
+            return list(json.loads(value_str))
```

Comment:
Won't this produce a list inside a list? I don't understand how this is working.

Reply:
No. The `list(...)` is consolidating to list type, apparently. Take a look:
![image](https://github.com/HathorNetwork/hathor-core/assets/5992210/7d8496a4-eb5b-4f8e-8d94-e19b2eba814b)

Reply:
I guess this solution would not work if the method returns a `tuple[int, list[str]]`, for instance. But I haven't really tested against this case yet.